st2common/st2common/openapi.yaml

```diff
@@ -8,7 +8,7 @@ info:
   version: "1.0.0"
   title: StackStorm API
```

Comment:
This file is automatically generated from `openapi.yaml.j2` one.

Please just make the change to the `.j2` file, run `make generate-api-spec` task and add and commit the generated file.

This will fix the tests / check which is currently failing and preventing us to merge this.

Reply:
Ok thanks for the info. I updated my commit.